src/note_encryption/orchard_domain.rs

```diff
@@ -1,79 +0,0 @@
-//! The OrchardDomain trait represents the difference between the `OrchardVanilla` and the `OrchardZSA`
-//! encryption and decryption procedures.
```

Comment:
need to make sure this file is seen as "moved" by git. not deleted.

Reply:
I think it is not possible. It automatically considered it as deleted because we have more than 50% of modifications in this file. It is due to the long comment of the function update_hash_with_actions.
You could see that in [my commit](https://github.com/QED-it/orchard/pull/121/commits/b1a56c421c9e1ece56322de15ec8ecb7fabf7e79) to rename the folder note_encryption into domain, it is considered as a move.